waspc/data/Generator/templates/sdk/wasp/client/crud/operationsHelpers.ts

```diff
@@ -0,0 +1,20 @@
+import { useAction, useQuery } from "../operations"
+import type { Tail } from "../../universal/types"
+
+// PRIVATE API
+export function makeUseQueryFor<Input, Output>(
+  query: Parameters<typeof useQuery<Input, Output>>[0]
```

Comment:
Please express this without querying the `useQuery` function (something like, `query: Query<Input, Output>`). We already have a type for this (if not, merge main into your branch and you'll have it :).

Doing it this way flips the source of truth and is more complicated (we're defining the core type by querying the helper that uses it).